### DISCUSSION TITLE
Lowest brightness setting on Pocketbook Touch HD3

### DISCUSSION THREAD

**Discussion by orthand** [koreader/koreader#7640] [Q&A] [answered]
Hey there,

I've found several posts here on Github as well as mobileread.com concerning using the lowest possible brightness.

I read mostly when it's dark and my very old Kobo H2O was pretty good at that. With my newer PB Touch HD3 even the night mode is too bright for my taste. I've tried taking a look at the files in /sys/class/backlight/ and actual_brightness. though it always tells me "5" using the text editor so I'm not too sure about how helpful that actually is.

Is there a way to further lower the brightness? (Will I need additional root access as standard PC link of course only lets me into /mnt/ext1/)
Would be glad about any idea or direction/link as my search results so far were rather scarce!

**Comment by pazos** [comment, marked as answer]
Nope. We use pocketbook APIs to request brightness. See https://github.com/koreader/koreader/blob/master/frontend/device/pocketbook/powerd.lua

The max brightness is 100, the minimum is 1. 0 means disabled. There's no way to put a value between 0 and 1 using PB APIs. No idea about what's possible on rooted devices.

**Reply by orthand** [reply]
Ah alright, too bad, had hoped for something similar for PB as discussed in #4186. Thanks a lot for your answer!
